src/palindrom-js-listener.js

```diff
@@ -187,6 +192,7 @@
 
         client.addEventListener('patchreceived', this.onPatchReceived);
         client.addEventListener('patchsent', this.onPatchSent);
+        client.addEventListener('patch-applied', this.onPatchApplied);
```

Comment:
You need to remove this listener later, don't you?

Reply:
Right!